From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@github.com>
Date: Fri, 29 May 2020 13:47:46 -0400
Subject: revert Add IChromeAccessible

This was added in https://chromium-review.googlesource.com/c/chromium/src/+/2206224
but it breaks WOA builds because
third_party/win_build_output/midl/ui/accessibility/platform/arm64 does not exist.
The link above says that the new interface is behind a feature flag which is
disabled by default so it is safe to remove for now.

diff --git a/ui/accessibility/BUILD.gn b/ui/accessibility/BUILD.gn
index c1b07e18ae6ac6a5b58e3caf9ed268c2f02fb313..476a3c55375c8c052e1719e6757d2072aa354d12 100644
--- a/ui/accessibility/BUILD.gn
+++ b/ui/accessibility/BUILD.gn
@@ -16,10 +16,6 @@ if (is_android) {
   import("//build/config/android/rules.gni")
 }
 
-if (is_win) {
-  import("//build/toolchain/win/midl.gni")
-}
-
 # Reset sources_assignment_filter for the BUILD.gn file to prevent
 # regression during the migration of Chromium away from the feature.
 # See docs/no_sources_assignment_filter.md for more information.
@@ -74,14 +70,6 @@ jumbo_component("ax_base") {
   ]
 }
 
-#if (is_win) {
-#  midl("ichromeaccessible") {
-#    sources = [
-#      "platform/ichromeaccessible.idl",
-#    ]
-#  }
-#}
-
 jumbo_component("accessibility") {
   defines = [ "AX_IMPLEMENTATION" ]
 
@@ -217,8 +205,7 @@ jumbo_component("accessibility") {
     ]
 
     public_deps += [
-      "//third_party/iaccessible2",
-      "//ui/accessibility/platform:ichromeaccessible",
+      "//third_party/iaccessible2"
     ]
 
     libs = [
@@ -374,8 +361,7 @@ test("accessibility_unittests") {
 
   if (is_win) {
     deps += [
-      "//third_party/iaccessible2",
-      "//ui/accessibility/platform:ichromeaccessible",
+      "//third_party/iaccessible2"
     ]
 
     libs = [
